index.js

```diff
@@ -39,35 +43,62 @@ function Heartbeat(baseUrl, pulse = 60) {
  * @param {HeartbeatTimer~callback}[callback] - optional
  */
 Heartbeat.prototype.pulse = function (host, category, type, name, threshold, callback) {
-  
+
   if (typeof threshold === 'function') callback = threshold;
 
+
+  var beatId = host + '-' + category + '-' + type + '-' + name;
+  var lastPulse = _.has(this._lastPulse, beatId) ? this._lastPulse[beatId] : 0;
+  var thresholdHrs = 25
+  if (category === "manfred") {
```

Comment:
why this?

Reply:
zelda funtionality. If threshold is 0 then we set a flag on the heartbeat. manfred is one of the "anti-heartbeats", which is raised actively.